src/test/java/uk/gov/hmcts/ccd/endpoint/std/CaseDetailsEndpointUserRolesIT.java

```diff
@@ -180,44 +168,12 @@ public void shouldReturn404WhenPostCreateCaseWithNoCreateCaseAccessOnCreatorRole
             CASE_TYPE_CREATOR_ROLE_NO_CREATE_ACCESS);
     }
 
-    @DirtiesContext(methodMode = BEFORE_METHOD)
-    @Test
-    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = { "classpath:sql/insert_cases.sql" })
-    public void shouldReturnPaginatedSearchMetadataForCitizen() throws Exception {
```

Comment:
i remember this test wouldn't work without the dirties context and could never work out why, but are we ok to delete the test ?

Reply:
This is not part of this ticket - probably came in via CCD-913

Reply:
could you check with @rajaram-nagarajan  please

Reply:
This test was delete by changes cherry-picked from [CCD-913-develop](https://github.com/hmcts/ccd-data-store-api/blob/CCD-913-develop/src/test/java/uk/gov/hmcts/ccd/endpoint/std/CaseDetailsEndpointUserRolesIT.java)

I have reach out to @rajaram-nagarajan and expecting him to respond here on this thread

Reply:
Yes, this test would not work without dirties context. The same test is present in CaseDetailsEndpointIT.java as well. So, I decided to delete this duplicate one.